backend/typescript/services/implementations/adminService.ts

```diff
@@ -47,6 +49,61 @@ class AdminService implements IAdminService {
     }
     return waiverDto;
   }
+
+  async updateFormTemplate(form: FormTemplateDTO): Promise<FormTemplateDTO> {
+    let formTemplateDTO: FormTemplateDTO | null;
+    try {
+      /* eslint no-underscore-dangle: 0 */
+      const formQuestions: Array<FormQuestionDTO> = [];
+      await Promise.all(
+        form.formQuestions.map(async (formQuestion, i) => {
+          const question = await formQuestionModel.create({
+            type: formQuestion.type,
+            question: formQuestion.question,
+            required: formQuestion.required,
+            description: formQuestion.description,
+            options: formQuestion.options,
+          });
+          formQuestions[i] = question._id;
+        }),
+      );
+      await formTemplateModel.updateOne(
+        {
+          formQuestions: { $exists: true },
+        },
+        {
+          $set: { formQuestions },
+        },
+        { upsert: true, runValidators: true },
+      );
+      formTemplateDTO = await this.getFormTemplate();
+    } catch (error: unknown) {
+      Logger.error(
+        `Failed to update waiver. Reason: ${getErrorMessage(error)}`,
+      );
+      throw error;
+    }
+    return formTemplateDTO;
+  }
+
+  async getFormTemplate(): Promise<FormTemplateDTO> {
+    let formTemplateDTO: FormTemplateDTO | null;
+    let form: FormTemplateDTO | null;
+    try {
+      form = await formTemplateModel.findOne();
```

Comment:
you need to populate the formQuestions here so we return the full formQuestion objects and not just the ids; an example of populating here: https://github.com/uwblueprint/focus-on-nature/blob/main/backend/typescript/services/implementations/camperService.ts#L150